### PR TITLE
mimic: scrub warning check incorrectly uses mon scrub interval

### PR DESCRIPTION
**Pull request by pdvian**
http://tracker.ceph.com/issues/38243

### CODE REVIEW

File: PendingReleaseNotes [hunk 1]
```diff
@@ -1,3 +1,11 @@
+14.0.1
+------
```
Comment (smithfarm) [inline PendingReleaseNotes]: Please change 14.0.1 to 13.2.5 here.

**Comment by gregsfortytwo** [comment]
Hmm, while it's a better way to be, I'm not sure it's appropriate to remove these config values from a stable release. How do we feel about that?
Maybe we should convert between the old static values and the new ratios if the user configured a non-default value.

**Comment by dzafman** [comment]
@gregsfortytwo Shouldn't we have addressed your config suggestions in master, not here in the back port?

**Comment by gregsfortytwo** [comment]
@dzafman master was across major releases where it's reasonable to change the config options. I'm saying I'm not sure it is reasonable within a stable release.

**Comment by smithfarm** [comment]
Marking DNM based on @gregsfortytwo comments.

**Comment by smithfarm** [comment]
Removing the `mon_warn_not_scrubbed` and `mon_warn_not_deep_scrubbed` config options (by replacing them with differently named options with different semantics) seems like a potentially disruptive change that would warrant careful consideration for mimic, which is supposed to be "stable".

What will happen if someone has these options set in their ceph.conf and is relying on them, and then upgrades to a new point release that removes/replaces them? My suspicion is they will be more-or-less silently ignored and stop working?

**Comment by dzafman** [comment]
@smithfarm @gregsfortytwo ceph/ceph#26493 is an alternate version of this backport that doesn't change the config values nor the usage of them.  It includes the checking of the correct config interval, getting the pool interval if present, a modified test case to match and the other minor commits.

**Comment by gregsfortytwo** [comment]
Yeah, closing in favor of the non-config-change version #26493.

**Pull request closed without merging**